UI/app_update.ts

```diff
@@ -364,6 +366,40 @@ export async function* UI_Interaction_Update(
                 }
             }
             model.rightPanelModel.show = true;
+        } else if (event.type == "ViewNoteDetail") {
+            let root: NostrEvent;
+            root = event.event;
+            if (event.event.parsedTags.root) {
+                const res = model.app.eventSyncer.syncEvent(NoteID.FromHex(event.event.parsedTags.root[0]));
+                if (res instanceof Promise) {
+                    continue;
+                }
+                root = res;
+            } else if (event.event.parsedTags.e) {
+                const res = model.app.eventSyncer.syncEvent(NoteID.FromHex(event.event.parsedTags.e[0]));
+                if (res instanceof Promise) {
+                    continue;
+                }
+                root = res;
+            }
+
+            if (root.kind == NostrKind.TEXT_NOTE) {
+                model.navigationModel.activeNav = "Social";
+                model.social.focusedContent = root;
+            } else if (model.navigationModel.activeNav == "DM") {
```

Comment:
why not `root.kind == NostrKind.Directed_Message`?

Reply:
right